project/dexter-vd-cpp/src/java/com/samsung/sec/dexter/vdcpp/checkerlogic/USleepCheckerLogic.java

```diff
@@ -238,6 +239,21 @@ private PreOccurence createPreOccurence(AnalysisConfig config,
 				return preOcc;
 
 			}
+			
+			private Integer convertIntoDecimalNumber(String string) 
```

Comment:
If the string can have not only number but also alphabet,
please catch exception of NumberFormatException.
Please read this : https://docs.oracle.com/javase/7/docs/api/java/lang/Integer.html#valueOf(java.lang.String)